candle-core/src/display.rs

```diff
@@ -63,6 +63,17 @@ impl std::fmt::Debug for Tensor {
             DType::F32 => self.fmt_dt::<f32>(f),
             DType::F64 => self.fmt_dt::<f64>(f),
             DType::F8E4M3 => self.fmt_dt::<F8E4M3>(f),
+            DType::Quantized(_) => {
+                // Dequantize to f32 for display
+                match self.to_dtype(DType::F32) {
+                    Ok(f32_tensor) => f32_tensor.fmt_dt::<f32>(f),
+                    Err(_) => write!(
+                        f,
+                        "Quantized Tensor[dtype={}; error converting to f32]",
+                        self.dtype().as_str()
+                    ),
+                }
+            }
```

Comment:
Is this easier or harder to read? 🤔 
```suggestion
            DType::Quantized(_) => self
                .to_dtype(DType::F32)
                .map(|t| t.fmt_dt::<f32>(f))
                .unwrap_or_else(|_| {
                    write!(
                        f,
                        "Quantized Tensor[dtype={}; error converting to f32]",
                        self.dtype().as_str()
                    )
                }),
```